Katello/katello
# Fixes #35711 - Product details doesn't close when smart proxy details are expanded

#### What are the changes introduced in this pull request?
Hide product details when smart proxy details are expanded.

#### Considerations taken when implementing this change?

#### What are the testing steps for this pull request?

1. Create a simplified ACS.
2. In the details split view, click to expand Products
3. Click to expand Smart Proxies
4. Products should close.
